### PR TITLE
Pioneer 18 precruise SM deploy csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Pre-cruise Surface Mooring deploy csvs - these have dummy placeholders for deploy date/time, lat/lon, depth.
Also these have the CPM & DCL board assemblies that were on the last cross-mio PR (with different UID format using the RDB PN/SN combo - also captured on the config sheets in Vault).

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 18 AM Updates [DT-152]](https://trello.com/c/zFi7BfCv/152-pioneer-18-am-updates-dt-152)

File: deployment/CP01CNSM_Deploy.csv [hunk 1]
```diff
@@ -508,4 +508,38 @@ AR61,,,,CP01CNSM-SBD12-03-HYDGN0000,15,1,2021-11-03T14:45:00,,CGMCP-01CNSM-00015
...
+AR66,,,,CP01CNSM-SBD11-01-MOPAK0000,16,1,2022-04-08T12:00:00,,CGMCP-01CNSM-00016,,CGINS-MOPAK0-11843,40.1,-70.7,,0,132,,,
+AR66,,,,CP01CNSM-SBD11-02-HYDGN0000,16,1,2022-04-08T12:00:00,,CGMCP-01CNSM-00016,,CGINS-HYDGN0-42330,40.1,-70.7,,0,132,,,
+AR66,,,,CP01CNSM-SBD11-06-METBKA000,16,1,2022-04-08T12:00:00,,CGMCP-01CNSM-00016,,CGINS-METLGR-00010,40.1,-70.7,,0,132,,,
+AR66,,,,CP01CNSM-SBD12-00-DCLENG000,16,1,2022-04-08T12:00:00,,CGMCP-01CNSM-00016,,3718-00001-00001-00054,40.1,-70.7,,0,132,,,
+AR66,,,,CP01CNSM-SBD12-03-HYDGN0000,16,1,2022-04-08T12:00:00,,CGMCP-01CNSM-00016,,CGINS-HYDGN0-Z5971,40.1,-70.7,,0,132,,,
+AR66,,,,CP01CNSM-SBD12-04-PCO2AA000,16,1,2022-04-08T12:00:00,,CGMCP-01CNSM-00016,,CGINS-PCO2AA-31124,40.1,-70.7,,0,132,,,
+AR66,,,,CP01CNSM-SBD12-05-WAVSSA000,16,1,2022-04-08T12:00:00,,CGMCP-01CNSM-00016,,CGINS-WAVSSA-05781,40.1,-70.7,,0,132,,,
+AR66,,,,CP01CNSM-SBD12-06-METBKA000,16,1,2022-04-08T12:00:00,,CGMCP-01CNSM-00016,,CGINS-METLGR-00017,40.1,-70.7,,0,132,,,
+AR66,,,,CP01CNSM-SBD12-08-FDCHPA000,16,1,2022-04-08T12:00:00,,CGMCP-01CNSM-00016,,CGINS-FDCHPB-11802,40.1,-70.7,,-5,132,,,
```
Comment (reedan88) [inline deployment/CP01CNSM_Deploy.csv:545]: Looks good.

File: deployment/CP03ISSM_Deploy.csv [hunk 2]
```diff
@@ -433,4 +433,35 @@ AR61,,,,CP03ISSM-SBD11-02-HYDGN0000,14,1,2021-11-01T14:10:00,,CGMCP-03ISSM-00014
...
+AR66,,,,CP03ISSM-RID27-04-DOSTAD000,15,1,2022-04-08T12:00:00,,CGMCP-03ISSM-00015,,CGINS-DOSTAD-00389,40.3,-70.8,,7,93,,,
+AR66,,,,CP03ISSM-SBC11-00-CPMENG000,15,1,2022-04-08T12:00:00,,CGMCP-03ISSM-00015,,3703-00051-00001-00099,40.3,-70.8,,0,93,CPM secondary 3703-00051-00001-01079,,
+AR66,,,,CP03ISSM-SBD11-00-DCLENG000,15,1,2022-04-08T12:00:00,,CGMCP-03ISSM-00015,,3718-00001-00001-00046,40.3,-70.8,,0,93,,,
+AR66,,,,CP03ISSM-SBD11-01-MOPAK0000,15,1,2022-04-08T12:00:00,,CGMCP-03ISSM-00015,,CGINS-MOPAK0-43257,40.3,-70.8,,0,93,,,
+AR66,,,,CP03ISSM-SBD11-02-HYDGN0000,15,1,2022-04-08T12:00:00,,CGMCP-03ISSM-00015,,CGINS-HYDGN0-42316,40.3,-70.8,,0,93,,,
+AR66,,,,CP03ISSM-SBD11-06-METBKA000,15,1,2022-04-08T12:00:00,,CGMCP-03ISSM-00015,,CGINS-METLGR-00008,40.3,-70.8,,0,93,,,
+AR66,,,,CP03ISSM-SBD12-00-DCLENG000,15,1,2022-04-08T12:00:00,,CGMCP-03ISSM-00015,,3718-00001-00001-00069,40.3,-70.8,,0,93,,,
+AR66,,,,CP03ISSM-SBD12-03-HYDGN0000,15,1,2022-04-08T12:00:00,,CGMCP-03ISSM-00015,,CGINS-HYDGN0-41917,40.3,-70.8,,0,93,,,
+AR66,,,,CP03ISSM-SBD12-04-PCO2AA000,15,1,2022-04-08T12:00:00,,CGMCP-03ISSM-00015,,CGINS-PCO2AA-35260,40.3,-70.8,,0,93,,,
```
Comment (reedan88) [inline deployment/CP03ISSM_Deploy.csv:467]: Looks good!

File: deployment/CP04OSSM_Deploy.csv [hunk 3]
```diff
@@ -435,3 +435,34 @@ AR61,,,,CP04OSSM-SBD11-06-METBKA000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014
...
+AR66,,,,CP04OSSM-MFC31-00-CPMENG000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,3703-00051-00001-01076,39.9,-70.8,,444,445,,,
+AR66,,,,CP04OSSM-MFD35-00-DCLENG000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00056,39.9,-70.8,,444,445,,,
+AR66,,,,CP04OSSM-MFD35-01-ADCPSJ000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-ADCPSJ-21496,39.9,-70.8,,444,445,,,
+AR66,,,,CP04OSSM-MFD35-02-PRESFC000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-PRESFC-01400,39.9,-70.8,,444,445,,,
+AR66,,,,CP04OSSM-MFD35-04-VELPTB000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-VELPTB-09952,39.9,-70.8,,444,445,,,
+AR66,,,,CP04OSSM-MFD35-05-PCO2WB000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-PCO2WB-C0098,39.9,-70.8,,444,445,,,
+AR66,,,,CP04OSSM-MFD35-06-PHSEND000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-PHSEND-P0157,39.9,-70.8,,444,445,,,
+AR66,,,,CP04OSSM-MFD37-00-DCLENG000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00057,39.9,-70.8,,444,445,,,
+AR66,,,,CP04OSSM-MFD37-01-OPTAAD000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-OPTAAD-00159,39.9,-70.8,,,445,,,
```
Comment (reedan88) [inline deployment/CP04OSSM_Deploy.csv:446]: Missing depth

File: deployment/CP04OSSM_Deploy.csv [hunk 4]
```diff
@@ -435,3 +435,34 @@ AR61,,,,CP04OSSM-SBD11-06-METBKA000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014
...
+AR66,,,,CP04OSSM-RID27-04-DOSTAD000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-DOSTAD-00387,39.9,-70.8,,7,445,,,
+AR66,,,,CP04OSSM-SBC11-00-CPMENG000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,3703-00051-00001-01072,39.9,-70.8,,0,445,CPM secondary 3703-00051-00001-01073 ,,
+AR66,,,,CP04OSSM-SBD11-00-DCLENG000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00014,39.9,-70.8,,0,445,,,
+AR66,,,,CP04OSSM-SBD11-01-MOPAK0000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-MOPAK0-42298,39.9,-70.8,,0,445,,,
+AR66,,,,CP04OSSM-SBD11-02-HYDGN0000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-HYDGN0-49886,39.9,-70.8,,0,445,,,
+AR66,,,,CP04OSSM-SBD11-06-METBKA000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-METLGR-00037,39.9,-70.8,,0,445,,,
+AR66,,,,CP04OSSM-SBD12-00-DCLENG000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00051,39.9,-70.8,,0,445,,,
+AR66,,,,CP04OSSM-SBD12-03-HYDGN0000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-HYDGN0-49857,39.9,-70.8,,0,445,,,
+AR66,,,,CP04OSSM-SBD12-04-PCO2AA000,15,1,2022-04-08T12:00:00,,CGMCP-04OSSM-00015,,CGINS-PCO2AA-34233,39.9,-70.8,,0,445,,,
```
Comment (reedan88) [inline deployment/CP04OSSM_Deploy.csv:468]: Looks good.

**Review by reedan88 — CHANGES_REQUESTED** [review]
I spotted only one error with a missing deployment depth for OSSM MFN OPTAA

File: deployment/CP01CNSM_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/CP01CNSM_Deploy.csv:545]: looks good

File: deployment/CP03ISSM_Deploy.csv (hunk 2 above)
Comment (cdobs) [inline deployment/CP03ISSM_Deploy.csv:467]: looks good

**Review by reedan88 — APPROVED** [review]
Looks good! Thanks for the fix.

File: deployment/CP04OSSM_Deploy.csv [hunk 5]
```diff
@@ -435,3 +435,34 @@ AR61,,,,CP04OSSM-SBD11-06-METBKA000,14,1,2021-10-30T20:10:00,,CGMCP-04OSSM-00014
...
```
Comment (cdobs) [inline deployment/CP04OSSM_Deploy.csv:468]: looks good

**Review by cdobs — APPROVED** [review]
All pre-deployment configs look good

**Pull request merged**